src/guide/command-line-options.md

```diff
@@ -176,6 +176,41 @@ Currently, `--map-source-class-to-test` supports the only one strategy of mappin
 
 Under the hood, it builds a regex for `--filter` option: `--filter='FooTest|BarTest'`.
 
+### `--id`
```

Comment:
should we have something like "Available in infection 0.29.15+" (or whatever the next version will be)

Reply:
phpstan has a similar thing, see e.g.

<img width="723" alt="grafik" src="https://github.com/user-attachments/assets/3ce051e7-0ed4-41c7-ac80-1b1bad540f19" />

https://phpstan.org/config-reference#pollutescopewithblock

Reply:
Thanks, good idea. Added:

light theme:

<img width="248" alt="image" src="https://github.com/user-attachments/assets/8507c6e8-7ec5-4d66-a056-5e992d8b16bd" />

Dark theme:

<img width="222" alt="image" src="https://github.com/user-attachments/assets/8c12b9a6-696b-4e3f-bea1-59c31b332828" />

Reply:
btw, thank AI with helping me writing all those CSS 🤣